### PR TITLE
Image setting documentation

### LINKED ISSUE #2632: Improve image setting behavior and documentation
The image settings at the press and journal level are hard to understand and use. For example, the press override image (bottom left in the OLH theme) only works if it's an SVG. Otherwise, the image defaults to either the Janeway logo or a press-wide SVG.

- [x] Update setting descriptions to explain requirements
- [x] Update documentation

### PR DESCRIPTION
**Pull request by joemull**
Fixes #2632 with the exception of allowing images other than SVGs

### CODE REVIEW

**Comment by joemull** [comment]
@ajrbyers note that some of the help text is not showing up on the image settings page. The two that aren't showing up, `journal_models.thumbnail_image` and `journal_models.press_image_override`, are structured differently as ForeignKey rather than ImageField, which may have something to do with it

**Review by mauromsl — APPROVED** [review]

File: src/templates/admin/core/manager/index.html [hunk 1]
```diff
@@ -16,16 +16,15 @@
     <div class="row expanded" data-equalizer>
         <div class="large-3 columns end">
             <div class="box" data-equalizer-watch>
-                <div class="title-area">
+							  <div class="title-area">
```
Comment (mauromsl) [inline src/templates/admin/core/manager/index.html:19]: SOmething went wrong with whitespace here

**Review by mauromsl — APPROVED** [review]

**Comment by joemull** [comment]
As this branch hasn't been closed yet I'm going to use it to make a few more improvements including linking to and lightly editing: https://janeway.readthedocs.io/en/v1.4-rc-5/styling/image_guidelines.html

**Comment by ajrbyers** [comment]
@joemull there are now conflicts on this branch - let me know when its ready for review.

**Comment by mauromsl** [comment]
Jenkins: Test this please

**Comment by mauromsl** [comment]
Jenkins: Test this please

**Comment by joemull** [comment]
Jenkins: Test this please

**Pull request merged**